docs/SRS/SRS.tex

```diff
@@ -338,6 +334,156 @@ \subsubsection{Authentication}
 	      \end{enumerate}
 \end{enumerate}
 
+\subsubsection{Work Orders}
+\begin{enumerate}[resume*=business_events]
+	\item An appointment has been scheduled
+	      \begin{enumerate}[VP\arabic*.]
+		      \item Viewpoint: Vehicle Owner
+		            \begin{enumerate}
+			            \item[] N/A
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Owner
+		            \begin{enumerate}
+			            \item The system shall create a work order
+			            \item The system shall populate the customer data and vehicle data from quote
```

Comment:
For this section do you want to say “the quote” and “the appointment”

Reply:
I think quote makes more sense here, the appointment is the actual event, but the quote has the details and pricing

Reply:
So basically merging these two lines?
```latex
\item The system shall populate the customer data and vehicle data from quote
\item The system shall populate the customer data and vehicle data from appointment if quote not
```

Reply:
adding the word "the"